maps/rent-burden/index.html

```diff
@@ -0,0 +1,128 @@
+        <meta name="twitter:card" content="summary">
+        <meta name="twitter:url" content="https://projects.scpr.org/maps/rent-burden/">
+        <meta name="twitter:domain" content="https://projects.scpr.org/">
+        <meta name="twitter:site" content="KPCC">
+        <meta name="twitter:title" content="How much of your income goes to rent?">
+        <meta name="twitter:description" content="For years, financial planners used that 30 percent threshold as a rule of thumb for renters: no more than 30 percent of your gross income should go to rent. But for many these days, with rents higher than they've ever been in Southern California, the 30 percent rule is a distant fantasy, and Southern Californian renters are having to adapt.">
+        <meta name="twitter:image:src" content="https://a.scpr.org/i/2b682d189f666a97507709e844de303c/116637-full.jpg">
+        <meta name="twitter:image:src" content="">
+        <!-- end meta -->
+
+        <!-- begin css -->
+        <link href="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.0.0/css/bootstrap.min.css" rel="stylesheet" />
+        <link href="https://projects.scpr.org/static-files/v3-dependencies/css/jquery-ui.1.10.3.css" rel="stylesheet" type="text/css" media="all" />
+        <link href="https://projects.scpr.org/static-files/css/jquery-alerts/jquery.alerts.css" rel="stylesheet" type="text/css" media="all" />
+        <link href="https://projects.scpr.org/static-files/v3-dependencies/css/main-foundation-style-v3.css" rel="stylesheet" />
+        <link href="css/leaflet.usermarker.css" rel="stylesheet" />
+        <link href="//cdnjs.cloudflare.com/ajax/libs/leaflet/0.6.4/leaflet.min.css" rel="stylesheet" />
+        <link href="css/jAlert-v3.css" rel="stylesheet" type="text/css" />
+        <link href="css/style.css" rel="stylesheet" type="text/css" />
+
+        <!-- end css -->
+
+        <!--[if IE]>
+            <style>
+                .data-container .data-visuals .jumbotron #form-controls button,
+                .data-container .data-visuals .jumbotron #reset button { height: 0px; margin: 10px 0px 0 0px; }
+            </style>
+        <![endif]-->
+
+        <!-- html5 shim and respond.js ie8 support of html5 elements and media queries -->
+        <!--[if lt IE 9]>
+            <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.2/html5shiv.js"></script>
+            <script src="https://cdnjs.cloudflare.com/ajax/libs/respond.js/1.3.0/respond.js"></script>
+            <link href="http://projects.scpr.org/static-files/v3-dependencies/respond-proxy/respond-proxy.html" id="respond-proxy" rel="respond-proxy" />
```

Comment:
Let's change the href to point to
`https://projects.scpr.org/static-files/v3-dependencies/respond-proxy/respond-proxy.html`